src/objective-c/GRPCClient/GRPCCall+Cronet.h

```diff
@@ -39,8 +39,7 @@
  */
 @interface GRPCCall (Cronet)
 
-+(void) setUseCronet:(BOOL) useCronet
-                    :(cronet_engine *)cronet_engine;
++(void)useCronet:(cronet_engine *)cronet_engine;
 
 +(void *)getCronetEngine;
```

Comment:
Method names that start with `get` have the specific convention of putting the return value in a pointer passed as a parameter. For the equivalent to Java getters, the naming convention is:

`+(cronet_engine *)CronetEngine;`